### DISCUSSION TITLE
CNN Prima News [CZ]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#456] [Show and tell]
### Channel ID

CNNPrimaNews.cz

### Stream URL

https://prima-ott-live-sec.ssl.cdn.cra.cz/qFaY_2Lvzwe14me4RXup7A==,1670842346/channels/prima_cnn/playlist-live_lq-live_mq-live_hq.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Already in the list: please close.

**Comment by freearhey** [comment]
https://prima-ott-live-sec.ssl.cdn.cra.cz/qFaY_2Lvzwe14me4RXup7A==,1670842346/channels/prima_cnn/playlist-live_lq-live_mq-live_hq.m3u8 (offline)
<img width="829" alt="image" src="https://user-images.githubusercontent.com/7253922/232247694-875484e5-f305-4f95-80de-fd3880f30df4.png">

**Comment by Sphinxroot** [comment]
https://cnn.iprima.cz/vysilani#0-dnes

https://prima-ott-live-sec.ssl.cdn.cra.cz/1UMRdOwNoLgusOi9Dwom3A==,1686398990/channels/prima_cnn/playlist/cze/live_lq.m3u8

**Comment by freearhey** [comment]
https://prima-ott-live-sec.ssl.cdn.cra.cz/1UMRdOwNoLgusOi9Dwom3A==,1686398990/channels/prima_cnn/playlist/cze/live_lq.m3u8 (offline)
<img width="841" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/ea0c0762-45ef-4b5a-ae90-adc6c2d14dcf">

**Comment by Sphinxroot** [comment]
hum might be not 24/7  switch stream 
now  i get 
https://prima-ott-live-sec.ssl.cdn.cra.cz/Op46zBxPF900AaQDzPgaFQ==,1686589012/channels/prima_cnn/playlist-live_lq.m3u8

**Comment by kaikyoofers** [comment]
> hum might be not 24/7 switch stream now i get https://prima-ott-live-sec.ssl.cdn.cra.cz/Op46zBxPF900AaQDzPgaFQ==,1686589012/channels/prima_cnn/playlist-live_lq.m3u8

Link does not work
